packages/cloud-cognitive/src/components/CreateTearsheet/preview-components/MultiStepWithSectionsTearsheet.js

```diff
@@ -80,6 +87,18 @@ export const MultiStepWithSectionsTearsheet = () => {
         viewAllToggleOnLabelText="On"
         includeViewAllToggle>
         <CreateTearsheetStep
+          onMount={async () => {
+            try {
+              let data = await fetch('https://randomuser.me/api/?results=5');
```

Comment:
i know this is just for storybook, but 2 things- we really should never use let and just to be safe and avoid possible unknown errors you should be doing a check for `data.ok`. i'd personally refactor to
```
const data = await fetch('https://randomuser.me/api/?results=5');
const json = await data.json();
if (!data.ok) throw new Error('received non 200 response');
setAllTopicOwners(data.results);
```